plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -356,8 +388,8 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             int curpos = HIWORD(wParam);
 
             switch (wmEvent) {
-                //case TB_ENDTRACK:
-                //case TB_THUMBPOSITION:
+                case TB_ENDTRACK:
+                case TB_THUMBPOSITION:
```

Comment:
Explain these changes?

Reply:
Enabling those fixes a bug where the GUI values for volume/latency aren't updated when using the scroll wheel.

Reply:
You should mention this in your commit message (actually, I'd suggest you use a separate commit for this bug fix).

TB_THUMBPOSITION should also be placed where TB_THUMBTRACK is since you don't need to send a TBM_GETPOS message for it (see https://msdn.microsoft.com/en-us/library/windows/desktop/gg430018(v=vs.85).aspx).